e2e/pages/adf/content_services/search/components/numberRangeFilterPage.ts

```diff
@@ -0,0 +1,163 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import Util = require('../../../../../util/util');
+import { by, browser, protractor } from 'protractor';
+
+export class NumberRangeFilterPage {
+
+    fromInput = by.css('input[data-automation-id="number-range-from-input"]');
+    toInput = by.css('input[data-automation-id="number-range-to-input"]');
+    applyButton = by.css('button[data-automation-id="number-range-btn-apply"]');
+    clearButton = by.css('button[data-automation-id="number-range-btn-clear"]');
+    fromErrorInvalid = by.css('mat-error[data-automation-id="number-range-from-error-invalid"]');
+    fromErrorRequired = by.css('mat-error[data-automation-id="number-range-from-error-required"]');
+    toErrorInvalid = by.css('mat-error[data-automation-id="number-range-to-error-invalid"]');
+    toErrorRequired = by.css('mat-error[data-automation-id="number-range-to-error-required"]');
+    filter;
+
+    constructor(filter) {
+        this.filter = filter;
+    }
+    clearFromField() {
+        Util.waitUntilElementIsClickable(this.filter.element(this.fromInput));
+        this.filter.element(this.fromInput).getAttribute('value').then((value) => {
+            for (let i = value.length; i >= 0; i--) {
+                this.filter.element(this.fromInput).sendKeys(protractor.Key.BACK_SPACE);
+            }
+        });
+        return this;
+    }
+    checkFromFieldIsEmpty() {
+        browser.controlFlow().execute(async () => {
+            await expect(this.getFromNumber()).toEqual('');
```

Comment:
Usually the expects are part of the test. Could you use this.getFromNumber() and have this expect in the test?